app/src/interfaces/input-rich-text-md/input-rich-text-md.vue

```diff
@@ -1,10 +1,15 @@
 <template>
 	<div ref="markdownInterface" class="interface-input-rich-text-md" :class="view[0]">
 		<div class="toolbar">
-			<v-menu show-arrow placement="bottom-start">
+			<v-menu
+				v-if="view[0] !== 'preview' && toolbar.indexOf('heading') !== -1"
```

Comment:
We can probably use a `<template v-if="view[0] !== 'preview' ">` to wrap all these toolbar buttons (until before the spacer div) so we don't need to add `view[0] !== 'preview'` to every v-if 👍

As for `toolbar.indexOf('heading') !== -1`, technically it's true `indexOf` has better browser support, but maybe doing `toolbar.includes('heading')` is slightly more readable. Plus we don't support IE anymore 😉